hoomd/md/external/wall.py

```diff
@@ -0,0 +1,577 @@
+        })
+        super()._attach()
+
+    @property
+    def walls(self):
+        """`list` [`hoomd.wall.WallGeometry`]: \
+            The walls associated with this wall potential."""
+        return self._walls
+
+    @walls.setter
+    def walls(self, wall_list):
+        if self._walls is wall_list:
+            return
+        self._walls = hoomd.wall._WallsMetaList(wall_list, _to_md_cpp_wall)
+
+
+class LJ(WallPotential):
+    r"""Lennard-Jones wall potential.
+
+    Args:
+        walls (`list` [`hoomd.wall.WallGeometry` ]): A list of wall definitions
+            to use for the potential.
+
+    Wall force evaluated using the Lennard-Jones potential.  See
+    `hoomd.md.pair.LJ` for force details and base parameters and `WallPotential`
+    for generalized wall potential implementation.
+
+    Example::
+
+        walls = [hoomd.wall.Sphere(radius=4.0)]
+        lj = hoomd.md.external.wall.LJ(walls=walls)
+        lj.params['A'] = {"sigma": 1.0, "epsilon": 1.0, "r_cut": 2.5}
+        lj.params[['A','B']] = {"epsilon": 2.0, "sigma": 1.0, "r_cut": 2.8}
+        lj.params["A"] = {"r_extrap": 1.1}
+
+    .. py:attribute:: params
+
+        The potential parameters per type. The dictionary has the following
+        keys:
+
+        * ``epsilon`` (`float`, **required**) -
+          energy parameter :math:`\varepsilon` :math:`[\mathrm{energy}]`
+        * ``sigma`` (`float`, **required**) -
+          particle size :math:`\sigma` :math:`[\mathrm{length}]`
+        * ``r_cut`` (`float`, **required**) -
+          The cut off distance for the wall potential :math:`[\mathrm{length}]`
+        * ``r_extrap`` (`float`, **optional**) -
+          The distance to extrapolate the potential, defauts to 0
+          :math:`[\mathrm{length}]`
+          .
+
+        Type: `TypeParameter` [``particle_types``, `dict`]
+    """
+
+    _cpp_class_name = "WallsPotentialLJ"
+
+    def __init__(self, walls):
+
+        # initialize the base class
+        super().__init__(walls)
+
+        params = hoomd.data.typeparam.TypeParameter(
+            "params", "particle_types",
+            hoomd.data.parameterdicts.TypeParameterDict(epsilon=float,
+                                                        sigma=float,
+                                                        r_cut=float,
+                                                        r_extrap=0.0,
+                                                        len_keys=1))
+        self._add_typeparam(params)
+
+
+class Gauss(WallPotential):
+    r"""Gaussian wall potential.
+
+    Args:
+        walls (`list` [`hoomd.wall.WallGeometry` ]): A list of wall definitions
+            to use for the potential.
+
+    Wall force evaluated using the Gaussian potential.  See
+    `hoomd.md.pair.Gauss` for force details and base parameters and
+    `WallPotential` for generalized wall potential implementation.
+
+    Example::
+
+        walls = [hoomd.wall.Sphere(radius=4.0)]
+        # add walls to interact with
```

Comment:
```suggestion
        # add walls
```

Reply:
I suggest removing the comment entirely. The comment is irrelevant to the construction of a wall potential class.